src/loader/bgw_launcher.c

```diff
@@ -48,12 +48,32 @@ typedef enum AckResult
 	ACK_SUCCESS,
 } AckResult;
 
+/* See state machine in README.md */
+typedef enum SchedulerState
+{
+	/* Scheduler should be started but has not been allocated or started */
+	ENABLED = 0,
+	/* The scheduler has been allocated a spot in timescaleDB's worker counter */
+	ALLOCATED,
+	/* Scheduler has been started */
+	STARTED,
+	/* Scheduler is stopped and should not be started automatically. START and RESTART messages can re-enable the scheduler. */
+	DISABLED
+} SchedulerState;
+
 #ifdef TS_DEBUG
 #define BGW_LAUNCHER_RESTART_TIME 0
 #else
 #define BGW_LAUNCHER_RESTART_TIME 60
 #endif
 
+#ifdef TS_DEBUG
+#define BGW_LAUNCHER_POLL_TIME 10L
```

Comment:
I prefer time unit defines to give info on the unit, which also makes it implicitly known that it is a time thing.

For instance, `BGW_LAUNCHER_POLL_MILLIS` or `_MS` is much more informative I think since the current naming gives no indication of how long time this really is.

Reply:
changed to BGW_LAUNCHER_POLL_MS